integration_test/deployment-dr-buckets.sh

```diff
@@ -11,12 +13,15 @@ export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
 
 #remove old files from bamboo as they throw error
 rm *.tf
-git clone --branch develop --single-branch https://github.com/nasa/cumulus-orca.git && cd integration_test
+git clone --branch ${bamboo_BRANCH_NAME} --single-branch https://github.com/nasa/cumulus-orca.git && cd integration_test
```

Comment:
Does ${bamboo_BRANCH_NAME} show output as 'develop'?

Reply:
Only if you are testing against develop.

Reply:
good to know about this feature